modin/pandas/concat.py

```diff
@@ -132,3 +144,34 @@ def concat(
         else:
             result_df.columns = new_idx
     return result_df
+
+
+def _determine_name(objs: list, axis):
+    """
+    Determine names of index after concatenation along passed axis
+
+    Parameters
+    ----------
+    objs : list of DataFrames or QueryCompilers
+        objects to concatenate
+
+    axis : int or str
```

Comment:
This should only be `int` if you are checking `if axis` otherwise the string will always evaluate to True if it is not empty.

Reply:
added converting `axis` from `str` to `int`